packages/pluggableWidgets/carousel-web/tests/e2e/objects/carousel.widget.ts

```diff
@@ -0,0 +1,27 @@
+// import page from "../../../../../../configs/e2e/src/pages/page";
```

Comment:
I think you can remove this

Reply:
I kept the tests folder for @leonardomendix.